content/blog/brulee-0-2-0/index.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+output: hugodown::hugo_document
+
+slug: brulee-0-2-0
+title: brulee 0.2.0
+date: 2022-09-26
+author: Max Kuhn
+description: >
+    Version 0.2.0 of brulee introduces learning rate schedulers. 
+
+photo:
+  url: https://unsplash.com/photos/wiTWDYLURr8
+  author: Alex Munsell
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, brulee]
+---
+
+We're thrilled to announce the release of [brulee](https://tidymodels.github.io/brulee/) 0.2.0. brulee contains several basic modeling functions that use the torch package infrastructure, such as: neural networks, linear regression, logistic regression, and multinomial regression. 
```

Comment:
"torch package infrastructure" sounds a bit awkward to me. Would something like

"bluree contains an R interface (or R wrappers) to several torch models such as ... "

describe brulee correctly?